eventlogger/src/main/java/gov/cms/ab2d/eventlogger/LoggableEvent.java

```diff
@@ -0,0 +1,10 @@
+package gov.cms.ab2d.eventlogger;
+
+import java.time.OffsetDateTime;
+
+/**
+ * Interface describing a loggable event
+ */
+public interface LoggableEvent {
+    boolean log(OffsetDateTime eventTime);
```

Comment:
Should `Event` really be able to log itself, or should we separate the concerns and delegate actual logging to `Logger` implementations?

Reply:
I'm sure we'll have at least two `Logger` implementation: one for Splunk, and the other for writing into a DB table.

Reply:
I'm thinking we could demand that each event implementation be able to serialize itself using an agreed-upon format (JSON), in which case this interface would just contain a single method `JSON toJSON()`. Thoughts?

Reply:
That's a good point. Maybe even New Relic.

Reply:
`getId` could probably also be declared in the interface, since it appears you're using `id` throughout.